src/test/java/hudson/security/LdapMultiEmbeddedTest.java

```diff
@@ -402,60 +400,55 @@ public void when_first_and_second_are_unavailable_and_ignorable_then_lookup_fail
                 allOf(containsString(FAILED_COMMUNICATION_WITH_LDAP_SERVER),
                         containsString(WILL_TRY_NEXT_CONFIGURATION),
                         containsString(INVALID_URL_PREFIX + planetExpress.getPort())),
-                CoreMatchers.<Throwable>instanceOf(DataAccessException.class)));
-        assertThat(log, recorded(Level.WARNING,
```

Comment:
I think the original intent here was to check that there actually was two records in the log.

Reply:
Perhaps that was the intent, but in fact it was just repeating an identical assertion.

Reply:
See original https://github.com/jenkinsci/ldap-plugin/blob/ff2c95a970df0c0446fe31d3b218c8bded827c2e/src/test/java/hudson/security/LdapMultiEmbeddedTest.java#L401-L410 which AFAICT was just a mistake.

Reply:
See #31 by @dwnusbaum. `when_first_and_second_are_unavailable_and_ignorable_then_no_login` sensibly verifies that we have messages from both `planetExpress` and `sevenSeas`. Yet `when_first_and_second_are_unavailable_and_ignorable_then_lookup_fails`, perhaps a modified copy of that test, redundantly asserts that we have a message from `planetExpress` only.

Reply:
I think it was a typo, the second assertion should have used `sevenSeas.getPort` to make sure that the plugin tried to check both servers.